### PR TITLE
Fix problem where we transition to `Open` when force closing before lock transaction reached finality

### PR DESCRIPTION
**Pull request by da-kami**

### CODE REVIEW

File: daemon/src/model/cfd.rs [hunk 1]
```diff
@@ -2570,6 +2570,22 @@ mod tests {
...
+            .dummy_open(dummy_event_id())
+            .dummy_commit();
+
+        let maker_short = Cfd::maker_short()
+            .dummy_open(dummy_event_id())
+            .dummy_commit();
+
+        let taker_event = taker_long.handle_lock_confirmed();
+        let maker_event = maker_short.handle_lock_confirmed();
+        assert_eq!(taker_event.event, CfdEvent::LockConfirmedAfterFinality);
```
Comment (thomaseizinger) [inline daemon/src/model/cfd.rs:2653]: Nit: Add a newline here.

**Review by thomaseizinger — APPROVED** [review]
Thank you!

**Review by klochowicz — APPROVED** [review]
Thanks, covering more corner-cases is always good :)

**Pull request merged**